pkg/virt-handler/seccomp/seccomp.go

```diff
@@ -0,0 +1,78 @@
+package seccomp
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/containers/common/pkg/seccomp"
+)
+
+// Install seccomp, kubeletRoot should be passed in format: /proc/1/root/var/lib/kubelet/
+func InstallPolicy(kubeletRoot string) error {
+	const errMsgFormat string = "failed to install default seccomp profile: %v"
+
+	installPath := filepath.Join(kubeletRoot, "seccomp/kubevirt")
+	if err := os.MkdirAll(installPath, 0700); err != nil {
+		return fmt.Errorf(errMsgFormat, err)
+	}
+
+	profileBytes, err := json.Marshal(defaultProfile())
+	if err != nil {
+		return fmt.Errorf(errMsgFormat, fmt.Errorf("internal failure: %v", err))
+	}
+
+	profilePath := filepath.Join(installPath, "kubevirt.json")
+	currentProfileBytes, err := os.ReadFile(profilePath)
+	if err != nil && !errors.Is(err, os.ErrNotExist) {
+		return fmt.Errorf(errMsgFormat, err)
+	}
+	if bytes.Equal(currentProfileBytes, profileBytes) {
+		return nil
+	}
+
+	if err := os.WriteFile(profilePath, profileBytes, 0700); err != nil {
+		return fmt.Errorf(errMsgFormat, err)
+	}
+
+	return nil
+}
+
+func defaultProfile() *seccomp.Seccomp {
+	profile := seccomp.DefaultProfile()
```

Comment:
Yes, in the future it would be great to just list all allowed syscalls and block others by default. This needs a little bit more work tho and extending the default profile seems good enough as starting point to me.
BTW while cri-o uses this profile it is not per se "cri-o`" profile.